server/asset/btc/btc.go

```diff
@@ -381,7 +381,7 @@ func (btc *Backend) InitTxSizeBase() uint32 {
 }
 
 // FeeRate returns the current optimal fee rate in sat / byte.
-func (btc *Backend) FeeRate() (uint64, error) {
+func (btc *Backend) FeeRate(_ context.Context) (uint64, error) {
```

Comment:
Tons of refactoring needed down to the `call` method to use the context with BTC. Fortunately with http posts (not websocket with autoreconnect), requests fail immediately when connection cannot be established.